### PR TITLE
Feature/update to 4.1.4 rebased

### PR DESCRIPTION
**Pull request by nicolai-nordic**

### CODE REVIEW

File: do_prebuild.js [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+    const bin_path = bin_path_cmd.stdout.trim();
+    const prebuild_path = path.join(bin_path, 'prebuild');
+
+    let prebuild_options = `--backend cmake-js -r ${options.runtime} --prepack "node do_prebuild.js --prepack-only"`;
+
+    if (options.include_regex) {
+        prebuild_options += ` --include-regex "${options.include_regex}"`;
+    }
+
+    // options.versions.forEach(version => {
```
Comment (kenr) [inline do_prebuild.js:51]: Remove?

File: do_prebuild.js [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+    if (options.arch) {
+        prebuild_options += ` --arch="${options.arch}"`;
+    }
+
+    if (options.all_versions) {
+        prebuild_options += ' --all';
+    }
+
+    let cmake_options = `-G="${options.generator}"`;
+/*
```
Comment (kenr) [inline do_prebuild.js:67]: This comment section can be removed

File: package.json [hunk 3]
```diff
@@ -44,29 +38,33 @@
...
     "change-case": "3.1.0",
-    "cmake-js": "5.1.1",
+    "cmake-js": "6.1.0",
     "crc": "^3.8.0",
-    "jszip": "^3.2.2",
+    "jszip": "^3.7.1",
     "nan": "^2.14.0",
-    "underscore": "^1.9.1"
+    "prebuild": "git://github.com/kenr/prebuild.git",
+    "node-abi": "*",
```
Comment (kenr) [inline package.json:49]: This is OK for you guys?

Comment (nicolai-nordic) [reply package.json:49]: Fine as a temporary solution I think, but we should probably figure out a better way of doing this.

Comment (nicolai-nordic) [reply package.json:49]: Actually we discussed this now and we decided that we should transfer ownership to `NordicPlayground`.

**Review by kenr — APPROVED** [review]

**Pull request merged**